src/index.html

```diff
@@ -13,6 +13,40 @@
     />
```

Comment:
Stylesheets with the `.scss` extension won't work directly in HTML. You should link to the compiled `.css` file instead.